### ISSUE TITLE
Don't force placeholder image format

### ISSUE THREAD

**Issue by ilicmarko** [nystudio107/craft3-imageoptimize#149]
Generated placeholder images have a forced `jpg` file format. Here is an example

![image](https://user-images.githubusercontent.com/2827249/59664918-3ae80080-91b2-11e9-9a0c-e92f49d0aed8.png)

This is a PNG image that has been converted to JPG and gets a black background. This "looks" wrong when it loads.

I find a bit of code that forces a JPG format.
https://github.com/nystudio107/craft-imageoptimize/blob/2a9b385fb2ba38a601b4a04d21f62a62d8594396/src/services/Placeholder.php#L286

Maybe this can be improved to don't force the format like on the transformations or the solution to this is to generate a silhouette?

**Comment by khalwat** [comment]
Yeah I wish there was a way to specify what color it'd use on the background in these cases.

I'll have to think about the implications of not forcing the image to be a JPEG.
